### PR TITLE
Writable stream spec fixes

### PR DESCRIPTION
**Pull request by tschneidereit**
Fixes a few spec bugs introduced in #655.

r? @tyoshino and @domenic.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -3595,12 +3595,12 @@ nothrow>WritableStreamDefaultControllerError ( <var>controller</var>, <var>e</va
...
     1. Let _readyPromiseIsPending_ be *false*.
     1. If _oldState_ is `"writable"` and !
-       WritableStreamDefaultControllerGetBackpressure(_stream_.[[writableStreamController]]) is *true*, set
+       WritableStreamDefaultControllerGetBackpressure(_controller_) is *true*, set
        _readyPromiseIsPending_ to *true*.
     1. Perform ! WritableStreamDefaultWriterEnsureReadyPromiseRejectedWith(_stream_.[[writer]], _e_,
        _readyPromiseIsPending_).
   1. Set _controller_.[[queue]] to a new empty List.
-  1. If _controller_ is *undefined* or _controller_.[[writing]] is *false* or _controller_.[[inClose]] is *false*,
+  1. _controller_.[[writing]] is *false* and _controller_.[[inClose]] is *false*,
```
Comment (ricea) [inline index.bs]: Missing "If".

Comment (tschneidereit) [inline index.bs]: Thanks, fixed.

**Comment by ricea** [comment]
lgtm

**Comment by tyoshino** [comment]
lgtm2. thanks for fixing

**Pull request merged**